compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -5207,7 +5214,7 @@ private BMapType getMapTypeForAnydataOrJson(BType type, SymbolEnv env) {
         return mapType;
     }
 
-    private BArrayType getArrayTypeForAnydataOrJson(BType type) {
+    private BArrayType getArrayTypeForAnydataOrJson(BType type, SymbolEnv env) {
```

Comment:
This change done for https://github.com/ballerina-platform/ballerina-lang/issues/34954